src/patterns/task-list-pages/index.md.njk

```diff
@@ -70,14 +74,43 @@ Use the following labels to describe the different states of a task:
 - 'Completed' (in blue) if the user has completed the task
 
 
-### Group tasks into sections
+### Group tasks into steps
 
-If your transaction involves lots of tasks, make it manageable by splitting it up into sections that represent stages in the process.
+If your transaction involves lots of tasks, make it manageable by splitting it up into steps that represent stages in the process.
 
-For example, you could group all tasks which help users find out if your service is right for them in a section called ‘Check before you start’.
+For example, you could group all tasks which help users find out if your service is right for them in a step called ‘Check before you start’.
 
 Where possible, allow users to complete tasks in any order. This will help them plan their time and complete sections as and when they can.
 
+### Marking tasks as completed
+
+Sometimes, it’s better to let the user decide when a task is completed.
+
+This can be helpful when a task involves:
+
+* some questions that are optional
+* writing a long answer (such as in a '[textarea](https://design-system.service.gov.uk/components/textarea/)')
```

Comment:
This link doesn't need the domain name

```suggestion
* writing a long answer (such as in a '[textarea](/components/textarea/)')
```

Reply:
Also, a very unimportant nitpick, I'm not sure whether the component name needs to be in quote marks?

Reply:
@lfdebrux good point, let's remove?